bugbot/rules/pending_sec_no_tracking.py

```diff
@@ -0,0 +1,94 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from libmozdata.bugzilla import Bugzilla
+
+from bugbot import utils
+from bugbot.bzcleaner import BzCleaner
+
+
+class SecurityApprovalTracking(BzCleaner):
+    def __init__(self, channel):
+        super().__init__()
+        self.channel = channel
+        if not self.init_versions():
+            return
+
+        self.version = self.versions[channel] if self.versions else None
+
+        self.extra_ni = None
+
+    def description(self):
+        return "Bugs with attachments pending security approval and incomplete tracking flags"
+
+    def handle_bug(self, bug, data):
+        # Assuming these bugs are bugs that do not have a status flag set to either "affected" or "unaffected"
+        bugid = str(bug["id"])
+        data[bugid] = {
+            "id": bugid,
+            "summary": bug["summary"],
+            "assignee": bug["assigned_to"],
+        }
+        self.add_auto_ni(
+            bugid,
+            {"mail": bug["assigned_to"], "nickname": bug["assigned_to_detail"]["nick"]},
+        )
+
+        return bug
+
+    def get_bugs(self, date="today", bug_ids=[], chunk_size=None):
+        bugs = super().get_bugs(date, bug_ids, chunk_size)
+
+        Bugzilla(
+            bugs.keys(),
+            include_fields=self.fields,
+            bughandler=self.handle_bug,
+            bugdata=bugs,
+        ).wait()
+
+        self.extra_ni = bugs
+        return bugs
+
+    def get_extra_for_needinfo_template(self):
+        return self.extra_ni
+
+    def columns(self):
+        return ["id", "summary", "assignee"]
+
+    def get_bz_params(self, date):
+        start_date, _ = self.get_dates(date)
+
+        self.status = utils.get_flag(self.version, "status", self.channel)
+        self.tracking = utils.get_flag(self.version, "tracking", self.channel)
+        self.fields = [
```

Comment:
I originally had it as an attribute because I needed to use it in `get_bugs`, but now that we are not using it anymore, I've converted to a local variable: 05e3e8a31bf61a8669960ae30854e9add294b40c